.golangci.yml

```diff
@@ -54,7 +54,14 @@ linters:
     - testpackage
     - scopelint # deprecated
     - golint # deprecated
-
+    - gci
```

Comment:
`gci` - please don't ignore it - seems like there is a bug in golangci and go1.18 so maybe better to go back to go1.17 for now and maybe a use golangci [v1.44.2](https://github.com/golangci/golangci-lint/releases/tag/v1.44.2)